src/main/resources/init_db.sql

```diff
@@ -1,20 +1,39 @@
 CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
 USE `taxi_service`;
 
-DROP TABLE IF EXISTS `manufacturers`;
-CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+DROP TABLE IF EXISTS `taxi_service`.`manufacturers`;
+CREATE TABLE `taxi_service`.`manufacturers` (
+    `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(225) NOT NULL,
+    `country` VARCHAR(225) NOT NULL,
+    `is_deleted` TINYINT NOT NULL DEFAULT 0,
+    PRIMARY KEY (`id`));
 
-DROP TABLE IF EXISTS `drivers`;
-CREATE TABLE `drivers` (
-                                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
-                                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+DROP TABLE IF EXISTS `taxi_service`.`drivers`;
```

Comment:
redundant explicit schema call, and the same for others. 
Overall I do not recommend changing the SQL statements/scripts if they were already used at some point, you can have checksums mismatch in the future, when you will be working with RDBMS, but that is a topic for the future
```suggestion
DROP TABLE IF EXISTS `drivers`;
```